llvm/lib/Target/AMDGPU/Utils/AMDGPUBaseInfo.cpp

```diff
@@ -624,7 +624,9 @@ bool isMAC(unsigned Opc) {
          Opc == AMDGPU::V_FMAC_LEGACY_F32_e64_gfx10 ||
          Opc == AMDGPU::V_FMAC_DX9_ZERO_F32_e64_gfx11 ||
          Opc == AMDGPU::V_FMAC_F16_e64_gfx10 ||
+         Opc == AMDGPU::V_FMAC_F16_t16_e64_gfx11 ||
```

Comment:
Is this function doing anything more than has tied operand? This should be moved to something on the instruction definition

Reply:
Yes I agree. I think it only takes care of the tied operand.